### PR TITLE
[Notifier] Add Mastodon bridge

### PR DESCRIPTION
**Pull request by qdequippe**
PR related https://github.com/symfony/symfony/pull/48101

### CODE REVIEW

**Review by OskarStark — COMMENTED** [review]
Please add a version added directive

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Needs `6.3` branch before merge

File: notifier.rst [hunk 1]
```diff
@@ -729,7 +734,7 @@ typical alert levels, which you can implement immediately using:
 
         # config/services.yaml
         services:
-            notifier.flash_message_importance_mapper: 
+            notifier.flash_message_importance_mapper:
```
Comment (OskarStark) [inline notifier.rst]: This change is gone after a rebase

Comment (qdequippe) [reply notifier.rst]: I have to rebase?

Comment (OskarStark) [reply notifier.rst]: I will do

**Pull request merged**